packages/extension/src/view/devtools/components/siteBoundaries/relatedWebsiteSets/insights/index.tsx

```diff
@@ -82,14 +122,26 @@ const Insights = () => {
                   </>
                 ) : (
                   <p className="mt-4">
-                    This site is the primary domain of the Related Website Sets.
+                    This site is the primary domain of the Related Website Set.
```

Comment:
nit: set -> sets

Reply:
`Set` is kept because the sentence is talking about a particular set.
Whether a site is the primary domain of a particular set.